topiary-config/src/language.rs

```diff
@@ -156,9 +156,9 @@ impl Language {
         };
 
         let language = unsafe {
-            let language_fn: Symbol<unsafe extern "C" fn() -> tree_sitter::Language> =
+            let language_fn: Symbol<unsafe extern "C" fn() -> *const()> =
```

Comment:
The formatter wants this to be formatted as:

```suggestion
            let language_fn: Symbol<unsafe extern "C" fn() -> *const ()> =
```